hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
> - [ ] Less naive VNodeData interface

Here, as you wish, we can be really straight but generics :

```tsx
export interface VNodeData {
  [key: string]: string | number | Function | null
               | undefined | Object //as everything is object this is like all : `any`
}
```

Or we can dive into the horrific (like did in @Types/React), add a type to every attrs :-1: 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2101